### PR TITLE
⚡️Create endpoints and ideal JSON output

### PR DESCRIPTION
**Pull request by FaazAbidi**
**Updates**
- We've two endpoints now: /get and /enrich with same param (url)
- get endpoint is returning an ideal JSON output according to the README description.
- tag_parser.py is updated with three main functions: og tags, twitter tags, and fallback
- Removed multithreading from app.py (no need now)
- Removed Streams
- [minor] fixed codefactor issues with artsy.py
- The data precedence is as follows:
      1. Open graph tags
      2. Twitter meta tags
      3. HTML title, img, and paragraph

### CODE REVIEW

**Comment by joelrojo** [comment]
@FaazAbidi @alihaiderrizvi Looking good so far guys! 

I was thinking that another fun way to do a fallback if there is no OG or Twitter tags would be to take a screenshot of the website and have that returned as the image (rather than the first image). I did some high level research and found these solutions:

https://github.com/maaaaz/webscreenshot
http://webscraping.com/blog/Webpage-screenshots-with-webkit/

What do you guys think?

cc: @hxghhhh

**Comment by FaazAbidi** [comment]
> @FaazAbidi @alihaiderrizvi Looking good so far guys!
> 
> I was thinking that another fun way to do a fallback if there is no OG or Twitter tags would be to take a screenshot of the website and have that returned as the image (rather than the first image). I did some high level research and found these solutions:
> 
> https://github.com/maaaaz/webscreenshot
> http://webscraping.com/blog/Webpage-screenshots-with-webkit/
> 
> What do you guys think?
> 
> cc: @hxghhhh

Great Idea, that's way better. We'll look into this, seems pretty easy with a python wrapper. The output image will be on Heroku's server, is there a way to host that image directly to Analogue's server?
 
Wild idea: Screenshots can also be used to train a ML model for detecting the medium and form of a webpage since the current code is not handling those very well and is limited. Potential project for  @alihaiderrizvi 😛

**Comment by joelrojo** [comment]
> Great Idea, that's way better. We'll look into this, seems pretty easy with a python wrapper. The output image will be on Heroku's server, is there a way to host that image directly to Analogue's server?

@FaazAbidi hmm wondering if there's a way to store this in a temporary URL of some sort. The way I'm handling images is very specific to Rails (using `ActiveStorage` to connect to the `Content` model), so not sure how to connect directly from this service. The rails endpoint opens the image and attaches/uploads via `ActiveStorage`, which then stores it in an S3 bucket.

File: constants.py [hunk 1]
```diff
@@ -17,7 +17,5 @@
 sentry_org='o197336'
 sentry_project='5230205'
 
-art_access_key = '10f5aecdb02f490e'
-art_skey = 'b305d6fad6431043' # skey : secret key
-art_api_url = 'https://www.wikiart.org/en/Api/2/login?'
-art_artwork_url = 'https://www.wikiart.org/en/api/2/PaintingSearch?term='
+apiflask_key = '103ef741c3b84a04b44e0b3d94e8685a'
+apiflash_url_to_image = 'https://api.apiflash.com/v1/urltoimage'
```
Comment (joelrojo) [inline constants.py:21]: @FaazAbidi Nice find! Where did you find this API?

**Review by joelrojo — APPROVED** [review]

**Pull request merged**